transaction/core/src/ring_signature/rct_bulletproofs.rs

```diff
@@ -153,7 +156,7 @@ impl SignatureRctBulletproofs {
     /// * `output_values_and_blindings` - Value and blinding for each output amount commitment.
     pub fn sign<CSPRNG: RngCore + CryptoRng>(
         message: &[u8; 32],
-        rings: &[Vec<(Address, Commitment)>],
+        rings: &[Vec<(CompressedRistrettoPublic, CompressedCommitment)>],
```

Comment:
This now takes compressed points, which can be read directly from the TxOuts in a transaction.